server/app/services/applicant/question/IdQuestion.java

```diff
@@ -39,6 +41,16 @@ protected ImmutableMap<Path, ImmutableSet<ValidationErrorMessage>> getValidation
     return ImmutableMap.of(getIdPath(), validateId());
   }
 
+  @Override
+  public ImmutableMap<Path, String> getJsonEntries() {
+    return applicantQuestion.getContextualizedScalars().keySet().stream()
+        .filter(path -> !Scalar.getMetadataScalarKeys().contains(path.keyName()))
+        .collect(
+            toImmutableMap(
+                path -> path,
+                path -> applicantQuestion.getApplicantData().readAsString(path).orElse("")));
+  }
+
```

Comment:
Above code is taken from the default case in:

https://github.com/civiform/civiform/blob/358ad3dbb6e67aacb91afb3ede60e32ac281ba26/server/app/services/applicant/ReadOnlyApplicantProgramServiceImpl.java#L450-L508